core-java-modules/core-java-streams-simple/src/test/java/com/baeldung/streams/minmaxbygroup/OrderProcessorUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.streams.minmaxbygroup;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class OrderProcessorUnitTest {
+  @Test
```

Comment:
let's break the line here between the class and Test annotation

Reply:
Done